docs/gatsby/api-reference/ocular-config.md

```diff
@@ -49,5 +62,5 @@ Each EXAMPLE entry has the following fields
 | ---            | ---      | ---   |
 | `componentUrl` | `String` | The filename of a React (`.jsx`) file |
 | `title`        | `String` | The title of example. |
-| `image`        | `String` | The icon for the bullet. Typically `images/<example-screenshot>.png`. Images are resolved in the `static` folder. |
+| `image`        | `String` | The icon for the bullet. Typically `images/<example-screenshot>.png`. Images are resolved in the `static` folder. (Right now only support png files) |
```

Comment:
This was a recent regression. I opened an ocular-gatsby issue on it. One possibly related change is that I bumped `sharp` from 0.22 to 0.22.1 to support Node 12. Maybe check if more recent fix is now available.